### ISSUE TITLE
Crash when Y part of WM_NORMAL_HINTS set to zero by bad apps

### ISSUE THREAD

**Issue by azrazalea** [sharplispers/clx#65]
This is an old bug. See https://lists.gnu.org/archive/html/stumpwm-devel/2011-04/msg00009.html
and a report that someone fixed it in some implementation http://sourceforge.net/tracker/?func=detail&aid=3292605&group_id=1355&atid=101355.

I'm running into it specifically with some steam games. Patching my local build with the patch from the first link but it seems like it is some kind of workaround. Is there a better fix?

**Comment by azrazalea** [comment]
I have a branch here that we can PR if we like the mailing list solution https://github.com/azrazalea/clx/tree/apply_divide_by_zero_fix

**Comment by azrazalea** [comment]
Another similar crash i'm getting on my other pc:

```
Caught 'The value
          1567771847
        is not of type
          (MOD 11)' at the top level. Please report this.
Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1003436D93}>
0: (XLIB::DECODE-WM-SIZE-HINTS #(1008 0 0 0 0 1918 1078 1918 1078 1567674275 32514 1346 ...))
1: (STUMPWM::GET-NORMALIZED-NORMAL-HINTS #<XLIB:WINDOW :0 1C00002>)
2: (STUMPWM::UPDATE-WINDOW-PROPERTIES #S(TILE-WINDOW "Sunless Sea" #x1C00002) :WM_NORMAL_HINTS)
3: (STUMPWM::HANDLE-EVENT :DISPLAY #<XLIB:DISPLAY :0 (The X.Org Foundation R11900000)> :EVENT-KEY :PROPERTY-NOTIFY :EVENT-CODE 28 :SEND-EVENT-P NIL :SEQUENCE 3537 :WINDOW #<XLIB:WINDOW :0 1C00002> :EVENT-WINDOW #<XLIB:WINDOW :0 1C00002> :ATOM :WM_NORMAL_HINTS :TIME 1346597 :STATE :NEW-VALUE)
4: ((FLET SB-THREAD::WITH-RECURSIVE-LOCK-THUNK :IN XLIB:PROCESS-EVENT))
5: ((FLET #:WITHOUT-INTERRUPTS-BODY-387 :IN SB-THREAD::CALL-WITH-RECURSIVE-LOCK))
6: (SB-THREAD::CALL-WITH-RECURSIVE-LOCK #<CLOSURE (FLET SB-THREAD::WITH-RECURSIVE-LOCK-THUNK :IN XLIB:PROCESS-EVENT) {7FFFF651724B}> #<SB-THREAD:MUTEX "CLX Event Lock" owner: #<SB-THREAD:THREAD "main thread" RUNNING {1003436D93}>> T NIL)
7: ((FLET SB-IMPL::TIMEOUT-BODY :IN XLIB:PROCESS-EVENT))
8: (XLIB:PROCESS-EVENT #<XLIB:DISPLAY :0 (The X.Org Foundation R11900000)> :HANDLER #<FUNCTION STUMPWM::HANDLE-EVENT> :TIMEOUT 0 :PEEK-P NIL :DISCARD-P NIL :FORCE-OUTPUT-P T)
9: ((FLET #:WITHOUT-INTERRUPTS-BODY-387 :IN SB-THREAD::CALL-WITH-RECURSIVE-LOCK))
10: (SB-THREAD::CALL-WITH-RECURSIVE-LOCK #<CLOSURE (FLET SB-THREAD::WITH-RECURSIVE-LOCK-THUNK :IN STUMPWM::DISPATCH-ALL) {7FFFF65176BB}> #<SB-THREAD:MUTEX "CLX Event Lock" owner: #<SB-THREAD:THREAD "main thread" RUNNING {1003436D93}>> T NIL)
11: ((FLET STUMPWM::DISPATCH-ALL :IN "/home/azra/git-repos/stumpwm/stumpwm.lisp") #<XLIB:DISPLAY :0 (The X.Org Foundation R11900000)>)
12: ((:METHOD IO-LOOP (STUMPWM::SBCL-IO-LOOP)) #<STUMPWM::SBCL-IO-LOOP {10059926E3}> :DESCRIPTION "StumpWM") [fast-method]
13: (STUMPWM::STUMPWM-INTERNAL-LOOP)
14: (STUMPWM::STUMPWM-INTERNAL ":0")
15: (STUMPWM ":0")
16: ((LAMBDA NIL :IN "/home/azra/git-repos/stumpwm/make-image.lisp"))
17: ((FLET #:WITHOUT-INTERRUPTS-BODY-90 :IN SB-EXT:SAVE-LISP-AND-DIE))
18: ((LABELS SB-IMPL::RESTART-LISP :IN SB-EXT:SAVE-LISP-AND-DIE))
```

Haven't found a fix for taht one quite yet.

**Comment by dkochmanski** [comment]
Fixed in #66.
